dags/sentinel_2/sentinel-2_gap_filler.py

```diff
@@ -95,29 +106,95 @@ def get_common_message_attributes(stac_doc: Dict) -> Dict:
 
     return msg_attributes
 
+def find_latest_report(update_stac: bool = False) -> str:
```

Comment:
Can't we share this code with the Landsat DAG?

Reply:
probably yes, I just didn't want to waste time once that this logic will be deleted soon.